core-tests/jvm/src/test/scala/zio/blocking/BlockingSpec.scala

```diff
@@ -3,45 +3,49 @@ package zio.blocking
 import java.util.concurrent.atomic.AtomicBoolean
 
 import zio.duration.Duration
-import zio.{ TestRuntime, UIO }
-
-final class BlockingSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRuntime {
-
-  def is =
-    "BlockingSpec".title ^
-      s2"""
-    Make a Blocking Service and
-    verify that
-      `effectBlocking` completes successfully $e1
-      `effectBlockingCancelable` completes successfully $e2
-      `effectBlocking` can be interrupted $e3
-      `effectBlockingCancelable` can be interrupted $e4
-    """
-
-  def e1 =
-    unsafeRun(effectBlocking(())).must_===(())
-
-  def e2 =
-    unsafeRun(effectBlockingCancelable(())(UIO.unit)).must_===(())
-
-  def e3 = {
-    val res = unsafeRun(effectBlocking(Thread.sleep(50000)).timeout(Duration.Zero))
-    res must beNone
-  }
-
-  def blocking(released: AtomicBoolean) =
+import zio.{ UIO, ZIOBaseSpec }
+import BlockingSpecUtil._
+import zio.test._
+import zio.test.Assertion._
+
+class BlockingSpec
+    extends ZIOBaseSpec(
+      suite("BlockingSpec")(
+        suite("Make a Blocking Service and verify that that")(
+          testM("effectBlocking` completes successfully") {
+            for {
+              io <- effectBlocking(())
```

Comment:
How about using `assertM` to make these tests even shorter? You wouldn’t need the for comprehension.